conda-forge.yml

```diff
@@ -1,11 +1,15 @@
-compiler_stack: comp7
-max_py_ver: '37'
-max_r_ver: '35'
-provider:
-  win: azure
-conda_forge_output_validation: true
+conda_build_tool: rattler-build
 github:
   branch_name: main
   tooling_branch_name: main
 conda_build:
-  pkg_format: '2'
+  error_overlinking: true
+conda_forge_output_validation: true
+build_platform:
+  osx_arm64: osx_64
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
+test: native
```

Comment:
Why not allow emulated here?

Reply:
Actually, let me check whether it really fail or whether this is a copy/paste mistake with another recipe.

Reply:
So indeed it fails with ` $SRC_DIR/conda_build.sh: line 10: $PREFIX/bin/python: Bad CPU type in executable`.

I will re-enable native only tests.

(the linux CI error is transient and not related to this PR/repo)

Reply:
You need to rerender after the change, but also, is this a rattler-build bug?

cc @wolfv

Reply:
@hadim did you remove the build logs somehow or did azure mess up again? I cannot find the failed run's logs to look at the failure.

Reply:
I did nothing (no permissions, anyway). I also noticed that sometime, PR builds disappears from Azure. I thought it was a known configuration to prevent accumulating too many logs.

Reply:
Let me re-trigger a fail build so you can check the logs.

Reply:
Here you are: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1186483&view=logs&j=1e5dc756-6df8-5bcc-2493-e24b3c6ab45e&t=e59419df-4268-5ef1-62bd-5ec3818134b5

Reply:
Thank you! I opened this issue on rattler-build since I think this is a bug there: https://github.com/prefix-dev/rattler-build/issues/1454